client/src/stories/secondLogo.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components/macro";
+
+export const SecondLogo = styled.img`
+  width: 175px;
+  display: grid;
+  grid-area: image;
+`;
+
+export const Header = styled.h1`
+  font-size: 2em;
```

Comment:
> -I could only change the font size of h1 in GlobalStyles with an !important. Any other recommendations?

This overwrites global styles. If you would like to use ` h1 {   font-size: 3.5em }`, as in global styles I recommend deleting this line of code

Reply:
Thanks! 🤩
It solved the problem!